phoebe/frontend/bundle.py

```diff
@@ -1,6 +1,7 @@
 import sys
 import subprocess
 import os
+import numpy as np
```

Comment:
this seems to be unused in this PR

Reply:
we get it from importing all from parameters, right?  (Not really a circular import... but importing `*` surely isn't the best, let's make a ticket for that to refactor and explicitly import things, but that definitely shouldn't happen here)